### ISSUE TITLE
Excel custom functions sample add-in not working

### ISSUE THREAD

**Issue by higearappdev** [OfficeDev/Excel-Custom-Functions#297]
I have been trying to setup an Excel custom functions add-in, but it is not working. These are the steps I took to setup the add-in

1. Run "yo office --projectType excel-functions --name funcy2 --host excel --ts True" (node: 14.18.1, npm: 6.14.15, yo: 4.3.0)

2. Convert project to Shared Runtime
    [manifest-webpack.zip](https://github.com/OfficeDev/Excel-Custom-Functions/files/7641938/manifest-webpack.zip)

 3. Run "npm run build"

4. Run "npm start"

After step 4 the add-in loads partially. The taskpane can be opened, and clicking "Run" in the taskpane works as expected. However custom functions are not loaded. An error message is shown that says "One or more add-ins failed to load a required resource." See screenshot below.

![Add-In](https://user-images.githubusercontent.com/11461602/144424248-fb02f437-2aef-4037-828d-9d77a3dc3c5e.png)

Additional information:

**Windows**
Edition Windows 10 Enterprise
Version 20H2
OS build 19042.1288
Experience Windows Feature Experience Pack 120.2212.3920.0

**Office**
Microsoft Excel for Microsoft 365 MSO (16.0.13127.21790) 32-bit

From the webpack config it seemed to me that no files are served from the dist folder, so I tried removing the dist folder from the custom functions URLs in the manifest. This did not change the outcome.

Any help would be greatly appreciated.

Regards,
Gerrit Hoogakker

**Comment by millerds** [comment]
Can you try clearing you add-in cache and trying again?  

https://docs.microsoft.com/en-us/office/dev/add-ins/testing/clear-cache

**Comment by higearappdev** [comment]
I should have mentioned that in my original post, but clearing cache is one of the first things I tried. It did not solve the issue.

**Comment by millerds** [comment]
Did you try the project before making your changes (skipping step 2)?

**Comment by higearappdev** [comment]
I tried your latest suggestion. Based on my Windows/Office version that will run the add-in in Edge original WebView as opposed to IE11. See here:

https://docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins

 Edge does not have a local loopback exemption, and unfortunately I do NOT have the necessary permissions at this time to add it as instructed here:

https://docs.microsoft.com/en-us/office/troubleshoot/office-suite-issues/cannot-open-add-in-from-localhost

I will try to get this exemption added through the proper channels and report back. This may take some time.

**Comment by higearappdev** [comment]
After the loopback exemption was added, I tried running the add-in. This resulted in a security certificate error, since I'm using a self-signed certificate that is not on the trusted list. The IE11 engine gives the option to ignore the security warning. The Edges engine does not, so I setup my own test server using plain http. Voila! Everything works, including custom funtions!

The next step I took was disabling https for the shared runtime as well. This solved the problem. 

Ultimately the problem was the untrusted certificate, which prevented the functions.json from loading. The taskpane worked because of the override option given in IE11.

Thank you for the suggestion Darren. It finally put me in the right track.
